scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -171,32 +270,47 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with library IDs of interest 
+# subset sce to only contain samples with IDs of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
 # remove original integrated object 
 rm(integrated_sce)
 ```
 
-```{r}
-# we are left with 3 from each subdiagnosis 
+
+We now have an updated SCE object that contains 6 samples that were obtained from a mix of ARMS and ERMS patients.
+We are particularly interested in identifying differentially expressed genes in tumor cell types between ARMS and ERMS.
+Before we do that we need to ensure that we have enough biological replicates from each group to set up our experiment. 
+It is imperative to consider good experimental design and ensure that we have enough biological replicates (at least > 3 for each group) when performing differential gene expression analysis.
+Here one sample, or group of cells collected together, from one piece of tissue is one biological replicate.
```

Comment:
This sentence is a bit confusing, but I think we can probably just leave it off.